### ISSUE TITLE
Error: Cannot find module 'arg'

### ISSUE THREAD

**Issue by Elaniobro** [nuxt-community/storybook#27]
### Version
@nuxtjs/storybook:  1.0.0
nuxt:  2.14.0

### Steps to reproduce
Set up basic install

### What is Expected?
story book to run

### What is actually happening?
```bash
yarn nuxt storybook
yarn run v1.22.4
$ /Users/elaniobro/Sites/mySite/node_modules/.bin/nuxt storybook
internal/modules/cjs/loader.js:796
    throw err;
    ^

Error: Cannot find module 'arg'
Require stack:
- /Users/elaniobro/Sites/mysite/node_modules/@nuxtjs/storybook/dist/cli.js
- /Users/elaniobro/Sites/mysite/node_modules/@nuxtjs/storybook/bin/nuxt-storybook.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.Module._load (internal/modules/cjs/loader.js:686:27)
    at Module.require (internal/modules/cjs/loader.js:848:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/Users/elaniobro/Sites/mysite/node_modules/@nuxtjs/storybook/dist/cli.js:9:31)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Module.require (internal/modules/cjs/loader.js:848:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/elaniobro/Sites/mysite/node_modules/@nuxtjs/storybook/dist/cli.js',
    '/Users/elaniobro/Sites/mysite/node_modules/@nuxtjs/storybook/bin/nuxt-storybook.js'
  ]
}

 FATAL  Failed to run command nuxt-storybook:                                                   10:10:18
Error: Command failed with exit code 1: nuxt-storybook


   ╭────────────────────────────────────────────────────────────╮
   │                                                            │
   │   ✖ Nuxt Fatal Error                                       │
   │                                                            │
   │   Failed to run command `nuxt-storybook`:                  │
   │   Error: Command failed with exit code 1: nuxt-storybook   │
   │                                                            │
   ╰────────────────────────────────────────────────────────────╯

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### nuxt config
```js
....
  /*
   ** Storybook configuration
   */
  storybook: {
    // Options
    port: 4000
  },
...

```

**Comment by farnabaz** [comment]
Fixed in [v1.1.0](https://github.com/nuxt-community/storybook/releases/tag/v1.1.0)

**Comment by productfrontenddeveloper** [comment]
@farnabaz  some problem
"@nuxtjs/storybook": "^4.0.2",
![CleanShot_2021-05-15_at_08 57 39](https://user-images.githubusercontent.com/38668796/118351774-763e0600-b566-11eb-8866-619c90767130.png)
